packages/near-fast-auth-signer-e2e-tests/tests/models/SignMultiChain.ts

```diff
@@ -22,8 +22,11 @@ type TransactionDetail = {
 class SignMultiChain {
```

Comment:
I think this is not the right solution, we can't expect the user to double click it. 

I was investigating this and looks like the error comes from the forms validation, as you can see on the video attached. I'm checking how to fix it. 

Idk why it only happens in Safari... very weird. 

https://github.com/user-attachments/assets/1501f7fd-ede8-481a-8654-317dfd080870

Reply:
I did some changes on the forms and now it's passing locally with a single click (have to test on the CI yet)

The failing tests are the new ones I introduced. I'm working on it. 

I will push the fix on my PR for the EVM function call. 

![Screenshot 2024-07-19 at 15 28 48](https://github.com/user-attachments/assets/e65c84de-f38c-43db-9ff9-3ee856719745)

Reply:
Hey @Pessina thanks for spotting that, looking forward to your solution. Although the problem here is just on the local form we setup for testing e2e, that is, triggering the local form input click with playwright, we don't have such problem on real implementations like the demo app.

Reply:
Yeah, agree, it's a problem only on the e2e app.

But it's better to fix the root problem instead of a quick fix. 

I will open the PR soon, I'm struggling a bit with my tests now haha

Reply:
Haha okay then.
Do you want me to close this PR then?

Reply:
Up to you. 

Do you wanna check my solution first, to see if you are happy with it?

Reply:
All right then. 
Let me wait.